### PR TITLE
Add modified date for AssetPathEntity

### PR DESCRIPTION
**Pull request by CaiJingLong**

### CODE REVIEW

**Comment by CaiJingLong** [comment]
@AlexV525  
Your Feature, you can verify it.

File: lib/src/filter/filter_options.dart [hunk 1]
```diff
@@ -73,6 +82,7 @@ class FilterOptionGroup {
     result["createDate"] = createTimeCond.toMap();
     result["updateDate"] = updateTimeCond.toMap();
     result['containsEmptyAlbum'] = containsEmptyAlbum;
+    result['containsPathModified'] = containsPathModified;
```
Comment (AlexV525) [inline lib/src/filter/filter_options.dart:85]: Lack of `this.containsPathModified = other.containsPathModified` in `merge` method.

**Review by AlexV525 — CHANGES_REQUESTED** [review]
Tested. Overall the feature is good to go on Android, but one place still need to be fix.

**Comment by CaiJingLong** [comment]
Regarding the parameter name, do you have any comments?  We can modify it before merging.
Once released, it will trigger a breaking change.

File: lib/src/filter/filter_options.dart [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import 'dart:convert';
 
+import 'package:photo_manager/photo_manager.dart';
```
Comment (AlexV525) [inline lib/src/filter/filter_options.dart:3]: Could we import the minimal reference?

**Comment by AlexV525** [comment]
> @AlexV525 Regarding the parameter name, do you have any comments? We can modify it before merging.

Maybe `lastModified`, `updateAt` or `modifiedAt` ? Can't tell which one is better cause IMO they are all the same.

**Comment by CaiJingLong** [comment]
Are there any suggestions for the field in filterOption?

**Comment by AlexV525** [comment]
`FilterOption` looks good. `FilterOptionGroup` seems can be refactored with the constructor, it's `copyWith` method also behaves slightly different from common copy with methods. For example:

```yaml
class FilterOptionGroup {
  FilterOptionGroup({
    FilterOption? imageOption,
    FilterOption? audioOption,
    FilterOption? videoOption,
    this.containsEmptyAlbum = false,
    this.containsPathModified = false,
  });
}
```

Also deprecated methods can be removed.

**Comment by CaiJingLong** [comment]
> `FilterOption` looks good. `FilterOptionGroup` seems can be refactored with the constructor, it's `copyWith` method also behaves slightly different from common copy with methods. For example:
> 
> ```dart
> class FilterOptionGroup {
>   FilterOptionGroup({
>     FilterOption? imageOption,
>     FilterOption? audioOption,
>     FilterOption? videoOption,
>     this.containsEmptyAlbum = false,
>     this.containsPathModified = false,
>   });
> }
> ```
> 
> Also deprecated methods can be removed.

The PR only add new feature.
I will open new pull request to update ctor.

**Pull request merged**